src/components/shell/shell.js

```diff
@@ -265,7 +266,9 @@ class Shell extends BaseComponent {
    * @memberof Shell
    */
   render(props: any): React$Element<any> {
-    let playerClasses = [style.player, style.skinDefault];
+    let playerClasses = [style.player, style.skinDefault,
+      `playkit-${this.player.env.os.name.replace(/ /g, "_")}`,
```

Comment:
We can add the prefix as global in webpack.config and add this as token here.
See here how we use it with _webpack.DefinePlugin_ 
We define it here: https://github.com/kaltura/playkit-js-ui/blob/afe561f9f7953e7f64208f3fc9cf2f0edbab3c80/webpack.config.js#L9-L13

and use it here: 
https://github.com/kaltura/playkit-js-ui/blob/afe561f9f7953e7f64208f3fc9cf2f0edbab3c80/src/index.js#L23-L24

so you can add another define in webpack.config ➡️ `const CSS_MODULE_PREFIX = 'playkit'` and use it in webpack config in the define plugin and also in the style loader config where we define the [localIdentName](https://github.com/kaltura/playkit-js-ui/blob/master/webpack.config.js#L69)